CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changelog
 
 ### Features
 
+- Implemented ellipsis when loading the number of transactions ([PR 1313](https://github.com/input-output-hk/daedalus/pull/1313))
```

Comment:
@daniloprates please remove this line and place the link to PR under:
```
- Improve performance of rendering huge lists of transactions and addresses
```

Reply:
✅